src/SchemaDesign/SchemaDesigner.tsx

```diff
@@ -0,0 +1,418 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { Schema, SchemaQuestion, SplitSchema, SchemaType, Step } from '@/types/interfaces'
+import { withTheme } from '@rjsf/core'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Divider from '@mui/material/Divider'
+import Paper from '@mui/material/Paper'
+import Stack from '@mui/material/Stack'
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import TextField from '@mui/material/TextField'
+import Typography from '@mui/material/Typography'
+import React, { useEffect } from 'react'
```

Comment:
Can we be consistent by destructuring the package and only pulling out the functionality we want to use? Saves us having to do `React.` all over the place.

This also applies to some of your other `tsx` files in this PR.